### DISCUSSION TITLE
DevMode Not enabling.

### DISCUSSION THREAD

**Discussion by WalkTheEarth** [3kh0/ext-remover#1324] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hi all!

I have discovered that my chromebook (dedede on v130) refuses to go onto devmode. after CTRL+D, it requests to go into Secure Mode. but then [Even if I say no to it] it refuses to let me continue, and ends up back into Secure Mode. is this a sign of keyrolling?

**Comment by akabutnicer** [comment, marked as answer]
now was this process executed via a personal Chromebook or a managed Chromebook?

**Reply by WalkTheEarth** [reply]
Managed.

**Reply by akabutnicer** [reply]
then this obviously isn't keyrolling, but rather a simple policy block 💀

**Reply by WalkTheEarth** [reply]
Thank you.

**Comment by schoolexploitkid** [comment]
its just that it doesn't work because its blocked by system policy

**Reply by WalkTheEarth** [reply]
Okay.
